### PR TITLE
OBGM-671 Unable to import locations through new import page

### PR DESCRIPTION
**Pull request by alannadolny**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/core/LocationService.groovy [hunk 1]
```diff
@@ -18,17 +18,16 @@ import org.apache.poi.hssf.usermodel.HSSFWorkbook
...
 import grails.plugins.csv.CSVMapReader
+import org.pih.warehouse.data.LocationDataService
 import org.pih.warehouse.importer.ImportDataCommand
 import util.ConfigHelper
 
 @Transactional
 class LocationService {
 
     GrailsApplication grailsApplication
-    def dataService
```
Comment (alannadolny) [inline grails-app/services/org/pih/warehouse/core/LocationService.groovy:28]: dataService was not used, so I removed it

File: grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy [hunk 2]
```diff
@@ -271,7 +271,7 @@ class LocationApiController extends BaseDomainApiController {
             throw new IllegalArgumentException("File cannot be empty")
         }
 
-        if (importFile.fileItem.contentType != "text/csv") {
+        if (importFile.contentType != "text/csv") {
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy:274]: Is this a verified API change for uploaded files? If so, let's include a link to that documentation in the PR and/or ticket.

File: grails-app/services/org/pih/warehouse/core/LocationService.groovy [hunk 3]
```diff
@@ -18,17 +18,16 @@ import org.apache.poi.hssf.usermodel.HSSFWorkbook
...
-    def productTypeService
-    def userService
-
+    UserService userService
+    LocationDataService locationDataService
```
Comment (jmiranda) [inline grails-app/services/org/pih/warehouse/core/LocationService.groovy:30]: Do we actually use this? If not, we can remove this too.

**Review by jmiranda — COMMENTED** [review]
See comments

File: grails-app/services/org/pih/warehouse/core/LocationService.groovy [hunk 4]
```diff
@@ -18,17 +18,16 @@ import org.apache.poi.hssf.usermodel.HSSFWorkbook
...
 import util.ConfigHelper
 
 @Transactional
 class LocationService {
 
     GrailsApplication grailsApplication
-    def dataService
-    def productTypeService
-    def userService
-
```
Comment (kchelstowski) [inline grails-app/services/org/pih/warehouse/core/LocationService.groovy:31]: how about those? are they used? You mentioned only `dataService` that is not used

**Review by drodzewicz — APPROVED** [review]

File: grails-app/services/org/pih/warehouse/core/LocationService.groovy (hunk 4 above)
Comment (alannadolny) [reply grails-app/services/org/pih/warehouse/core/LocationService.groovy:31]: @kchelstowski @jmiranda 
userService:
image
locationDataService:
image

File: grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy (hunk 2 above)
Comment (alannadolny) [reply grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy:274]: We changed it many times, there is a ticket for investigating the rest of those places with `fileItem`: [ticket](https://pihemr.atlassian.net/browse/OBGM-669?atlOrigin=eyJpIjoiMWY2MWNiMzgwMjFjNDMzMzg3ZjNjNTY0ZGQzYjgwMDgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

ImportFile is a type of MultipartFile - on this instance, we can call getContentType, getSize, getBytes, etc.
Docs: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/multipart/MultipartFile.html

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**